tests/lib/filters/test_osconfig_agent_filter.py

```diff
@@ -0,0 +1,77 @@
+import datetime
+
+from lib.log_processor import ProcessedLogEntry
+from lib.filters.osconfig_agent_filter import osconfig_agent_filter
+
+
+def mock_log_entry(
```

Comment:
A more idiomatic approach would be to make this a `@pytest.fixture()`.
You could then use `dataclass.replace` to update values for each test.

See [this test](https://github.com/ONSdigital/blaise-slack-alerter/blob/main/tests/lib/log_processor/log_types/test_gce_instance.py#L39) for an example.